modules/account-lib/test/unit/coin/polygon/transactionBuilder/walletInitialization.ts

```diff
@@ -0,0 +1,109 @@
+import should from 'should';
+import { TransactionType } from '@bitgo/sdk-core';
+import { getBuilder, Polygon } from '../../../../../src';
+import * as testData from '../../../../resources/polygon/polygon';
+
+describe('Polygon wallet initialization', function () {
+  let txBuilder: Polygon.TransactionBuilder;
+  const initTxBuilder = (): void => {
+    txBuilder = getBuilder('tpolygon') as Polygon.TransactionBuilder;
+    txBuilder.fee({
+      fee: '10000000000',
+      gasLimit: '6800000',
+    });
+    txBuilder.counter(1);
+    txBuilder.type(TransactionType.WalletInitialization);
+  };
+
+  describe('should build', () => {
+    it('an init transaction', async () => {
+      initTxBuilder();
+      console.log('txBuilder', txBuilder);
```

Comment:
please remove log lines from tests after using them to debug - we are currently spewing a bunch of useless logs on test runs for account-lib now.

CC @gianchandania @mmcshinsky-bitgo

Reply:
Sure Tyler, will remove these and will take care of this going forward. Sorry for inconvenience